hello_world/src/main.rs

```diff
@@ -22,13 +25,25 @@ async fn hello_world(
         .log_in(username, password, None, Some("ruma-example-client"))
         .await?;
 
-    let room_id = client
-        .send_request(get_alias::v3::Request::new(room_alias))
-        .await?
-        .room_id;
+    // Join the room.
     client
-        .send_request(join_room_by_id::v3::Request::new(room_id.clone()))
+        .send_request(join_room_by_id_or_alias::v3::Request::new(
+            room_id_or_alias.clone(),
+        ))
         .await?;
+
+    // Resolve the room alias to a room ID if necessary.
```

Comment:
No need for the extra request, the response of `join_room_by_id_or_alias` contains the room ID.

Reply:
Oh right, thanks.